internal/service/elasticache/serverless_cache.go

```diff
@@ -196,7 +196,13 @@ func (r *serverlessCacheResource) Schema(ctx context.Context, request resource.S
 							NestedObject: schema.NestedBlockObject{
 								Attributes: map[string]schema.Attribute{
 									"maximum": schema.Int64Attribute{
-										Required: true,
+										Optional: true,
+										PlanModifiers: []planmodifier.Int64{
+											int64planmodifier.RequiresReplace(),
+										},
+									},
+									"minimum": schema.Int64Attribute{
```

Comment:
What do you think of add a validator for the constraint aws imposes? `You can set a minimum limit from 1 GB.` and